src/Core/Core.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <RestoreProjectStyle>PackageReference</RestoreProjectStyle>
     
-    <TargetFrameworks Condition=" ' $(TargetFrameworks) ' == '  ' ">netstandard2.0;netstandard1.3</TargetFrameworks>
+    <TargetFrameworks Condition=" ' $(TargetFrameworks) ' == '  ' ">netcoreapp3.1;netstandard2.0;netstandard1.3</TargetFrameworks>
```

Comment:
Any reason for adding `netcoreapp3.1` since `netstandard2.0` is fully compatible?